packages/manager/apps/dedicated/client/app/account/user/user.routes.js

```diff
@@ -20,6 +20,10 @@ angular.module('UserAccount').config(
       },
       redirectTo: `${name}.method`,
       resolve: {
+        angularQr: /* @ngInject */ ($ocLazyLoad) =>
+          import('angular-qr').then((module) =>
```

Comment:
I don't see `angularQr` being declared as needed by the module like it used to be https://github.com/ovh/manager/pull/3996/files#diff-24e0955591a31a9b81232f4bfa927a3d31c3118725d09b1eee1b7d012646e1afL11
this has no impact ?

Reply:
we used to declare 'ja.qr' because it was required by https://github.com/janantala/angular-qr
BUT
we are importing angular-qr which isn't the same repo, see : https://www.npmjs.com/package/angular-qr

qa.jr is not needed, also angular-qr is now injected dynamically